src/Account.ts

```diff
@@ -62,15 +62,22 @@ export class Account {
       this.reachAccount = await this.reachStdLib.newAccountFromMnemonic(this.mnemonic);
     } else if (this.secretKey && !this.reachAccount) {
       this.reachAccount = await this.reachStdLib.newAccountFromSecret(this.secretKey);
-    } else if (this.networkAccount && this.signer && this.reachAccount == null && this.provider && (!this.mnemonic && !this.secretKey)) {
+    } else if (
+      this.networkAccount &&
+      this.signer &&
+      this.reachAccount == null &&
+      this.provider &&
+      !this.mnemonic &&
+      !this.secretKey
+    ) {
       await this.reachStdLib.setWalletFallback(
         await this.reachStdLib.walletFallback({
           providerEnv: this.providerEnv ? this.providerEnv : this.network,
           [this.signer]: this.provider,
         }),
       );
       this.reachAccount = await this.reachStdLib.connectAccount(this.networkAccount);
-    } else if (this.signer && !this.reachAccount && this.provider && (!this.mnemonic && !this.secretKey)) {
+    } else if (this.signer && !this.reachAccount && this.provider && !this.mnemonic && !this.secretKey) {
```

Comment:
Is there a reason this line changed?

Reply:
@AwolDes 
It changed from running `yarn format --write`